src/components/Layout/styles.module.scss

```diff
@@ -6,6 +6,14 @@
   padding-bottom: $heigth-footer;
   position: relative;
 
+  .header{
+    background-color: $white;
+    box-shadow: 0 8px 8px -8px $gray-shadow;
+    position: sticky;
```

Comment:
This is **NOT** wrong because the component works, but I think `position: fixed` is the better fit here, since the element never changes its position. You can read more about it [here](https://dev.to/suryawiguna/css-position-fixed-vs-sticky-5232)

Reply:
Good!! great Article